modules/claims_api/app/controllers/claims_api/v2/veterans/evidence_waiver_controller.rb

```diff
@@ -31,6 +45,41 @@ def find_bgs_claim!(claim_id:)
 
           raise
         end
+
+        def find_or_create_ews(bgs_claim)
+          ews = ClaimsApi::EvidenceWaiverSubmission.find_by(claim_id: bgs_claim[:bnft_claim_dto][:bnft_claim_id])
+
+          unless ews&.status&.in?(%w[submitted pending])
+            attributes = {
+              status: ClaimsApi::EvidenceWaiverSubmission::PENDING,
+              auth_headers: auth_headers,
+              cid: source_cid
```

Comment:
we need a `claim_id` added here to run during the create. so something along the lines of 
`claim_id: bgs_claim.dig(:bnft_claim_dto, :bnft_claim_id)`